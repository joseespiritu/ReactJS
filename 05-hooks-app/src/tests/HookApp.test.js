import React from "react";
import HookApp from "../HookApp";
import { shallow } from "enzyme";

describe("Pruebas sobre componente <HookApp/>", () => {
  test("Debe de renderizar el componmente correctamente", () => {
    const wrapper = shallow(<HookApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
