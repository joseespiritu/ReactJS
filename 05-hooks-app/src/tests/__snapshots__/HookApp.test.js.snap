// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre componente <HookApp/> Debe de renderizar el componmente correctamente 1`] = `
<div>
  <h1>
    Hook App
  </h1>
</div>
`;
